### ISSUE TITLE
Design System: Avatars

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#3934]
## Description

FIgma: https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=49-1966&t=F4xkGsIXsy1KEh5Z-0

## Acceptance Criteria
- The avatar component should be updated to reflect 3 new sizings: 16px, 24px, 32px. 
- Tooltip over Avatar components are **out-of-scope** for updating our Avatar component.

**Comment by Miaplacidus** [comment]
@zakhap The Avatar component now considers sizing. However, existing elements cannot be simply switched over to 16, 24, or 32 px without looking rather awkward. The current sizing on avatars is all over the place. Hence, I don't think this card has much value until @meeshlin has completed designs around elements containing avatars, i.e. I think avatars are a poor choice for rip-and-replace.

**Comment by CowMuon** [comment]
Can we either get a response to above comment or update on this please? Just looking at this it's unclear what the current status is.

**Comment by zakhap** [comment]
What places are you concerned with updating the sizing? 

As far as I can quickly tell, most are already to spec: 
- the user avatars for the thread metadata rows is 24px already
- user avatars on the for you page are 16px
- user avatar in the profile dropdown in the upper right hand corner is 16px,
- the profile page avatar is 160px,

**Comment by Miaplacidus** [comment]
I agree that in most instances, avatars are already one of the 3 sizes we've decided upon, but there are a couple cases where that doesn't quite work. 1. On the thread preview page, it is sometimes the case that only the avatar is shown (as opposed to both an avatar and a username). In these cases, the avatar size is sometimes decreased by 4 px. It does look somewhat oversized when left at 24 px in these cases, but I could see how someone else would think it looks fine at a larger size. Do we want to do away with the logic around decreasing the size and, instead, stick with 24 px there? 2. On the edit profile page, the upload avatar is 60 px. Forcing this to be a smaller size just looks odd.

**Comment by jnaviask** [comment]
cc @zakhap and @meeshlin re these two locations

**Comment by Miaplacidus** [comment]
Updated comment above to account for popover. @zakhap @meeshlin

**Comment by zakhap** [comment]
> 1. On the thread preview page, it is sometimes the case that only the avatar is shown (as opposed to both an avatar and a username)
We should definitely do away with the "size changing" logic + stick to 24px for the moment.

> 2. On the edit profile page, the upload avatar is 60 or 108 px.
I feel like this component (the avatar uploader) is out of scope for this PR!

> 3. The user popover makes use of the Avatar component, and it, too, is not always one of the standard sizes.
It is technically out of scope, so I'm fine ignoring it. _But_ if it's easy to standardize that as well, just update them to 32px. 

I will comment this on the PR as well, but I am ok with it going out in its current version!

**Comment by CowMuon** [comment]
Can @zakhap or @Miaplacidus please unpack "size changing logic" in this scope? (ie, was there an issue requesting size changing logic to be implemented? Was size-changing logic introduced in a PR but in hindsight decided against? Jake does not know what this is either when I just now asked.)
